k6/spike.js

```diff
@@ -0,0 +1,49 @@
+import { check, sleep } from 'k6';
+import exec from 'k6/execution';
+import http from 'k6/http';
+
+export const options = {
+	// Key configurations for spike in this section
+	stages: [
+		{ duration: '2m', target: 2000 }, // fast ramp-up to a high point
+		{ duration: '1m', target: 0 }, // quick ramp-down to 0 users
+	],
+};
+
+const baseUrl = 'http://localhost:3001';
+const cookieName = 'prompt-injection.sid';
+const vuCookieJar = (() => {
+	const cookieJars = {};
+	return {
+		get: (id) => cookieJars[id],
+		set: (id, jar) => (cookieJars[id] = jar),
+	};
+})();
+
+export default () => {
```

Comment:
Is this the same function as in load.js? If so, extract the function into its own module and use it in both test files.